signac/__main__.py

```diff
@@ -538,7 +538,8 @@ def _sig(st):
     except SchemaSyncConflict as error:
         _print_err(
             "WARNING: The detected schemas of the two projects differ! "
-            "Use --no-scema-check to skip the schema check.")
+            "Use --no-scema-check to skip the schema check."
```

Comment:
There is a typo and you need to add a space after the dot ("."), to properly separate the sentences, however I'd like to actually propose something else:
- [ ] Rename the option to `-m`/`--merge`.
- [ ] Convert the warning to a simple more positive and affirmative message:

> Merging of two projects with different schemas requires the -m/--merge option.

My reasoning for the different option name is that it expresses the same behavior in a more positive way. "--no-schema-check" implies something negative to me, I am skipping a check and are potentially doing something bad, whereas `--merge` implies that I am doing something proper, just explicitly. Similar to how `cp` requires the `-r` option for recursive copying. It's not a bad thing, you just need to give explicit permission, because the consequences are bad if you did not really intend to do that.

See also: https://devguide.python.org/documenting/#affirmative-tone

@vyasr You cool with that?

Reply:
Yes, I think that is a good idea. I would also be clear about exactly what merging means, i.e. that statepoints existing in the source that are not in the destination will be copied over. This clarification avoids any confusion with respect to the merge strategy for individual jobs.